views/admin.functions.subscribe.options.php

```diff
@@ -163,10 +163,6 @@ function field_type( $args ) {
 				  <option value="text"' . selected( esc_attr( $value ), 'text' ) . '>Text Field</option>
 				  <option value="tel"' . selected( esc_attr( $value ), 'tel' ) . '>Telephone</option>
 				  <option value="date"' . selected( esc_attr( $value ), 'date' ) . '>Date</option>
-				  <option value="hidden"' . selected( esc_attr( $value ), 'hidden' ) . '>Hidden</option>
-				  <option value="select"' . selected( esc_attr( $value ), 'select' ) . '>Select</option>
-				  <option value="radio"' . selected( esc_attr( $value ), 'radio' ) . '>Radio</option>
```

Comment:
both 'radio' and 'checkbox' types use 'radio' value match to check for selection, can you check if that is causing them not to work?